src/components/tables/DataTable.vue

```diff
@@ -75,6 +75,14 @@ export default {
       this.component.columns(colIndex)
         .search(event.target.value)
         .draw();
+    },
+    onKeydown: function(event) {
+      if (event.which === 65 && (event.ctrlKey || event.metaKey)) {
```

Comment:
By this logic, does it mean that Cmd + C/X/V also don't work? If so, why is it that we need to be writing our own event listeners for the native browser features?

Reply:
Actually everything except ctrl + a worked fine before. Now everything works fine.

Reply:
@MaksymShokin Thanks. I've approved the work. Could you leave a comment to explain why Ctrl + A didn't work so I can understand?

Reply:
@tonytlwu 
Actually I do know why  Ctrl + A did not work. Apparently, library has some issues with it. I did not find anything regarding it in the library docs. In the analytics Ctrl + A does not work also.